extensions/skyportal/static/js/components/Alerts.jsx

```diff
@@ -68,37 +97,324 @@ const useStyles = makeStyles((theme) => ({
   header: {
     paddingBottom: "0.625rem",
   },
+  button: {
+    textTransform: "none",
+  },
+  wrapperRoot: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  wrapper: {
+    margin: theme.spacing(1),
+    position: 'relative',
+  },
+  buttonProgress: {
+    color: theme.palette.text.secondary,
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    marginTop: -12,
+    marginLeft: -12,
+  },
+  grid_item_table: {
+    order: 1,
+    [theme.breakpoints.down('lg')]: {
```

Comment:
SkyPortal docs technically specify using only `breakpoints.up()`  [here](https://skyportal.io/docs/styling.html#responsive-design) so you should probably refactor this to have `order: 2` by default and switch to 1 at the `breakpoint.up` :)

Reply:
Damn, I always forget which one it should be... I first did `up` and then thought: "oh no, it should be `down`" and rewrite it 🤣. Fixed.